package.json

```diff
@@ -87,7 +87,7 @@
     "passport-http-bearer": "1.0.1",
     "passport-oauth2-client-password": "0.1.2",
     "path-match": "1.2.4",
-    "proxyquire": "^2.1.0",
+    "proxyquire": "2.1.0",
```

Comment:
Sorry i only just realised as well - this should be a dev dep :see_no_evil:

Reply:
Good catch!